test_radix.py

```diff
@@ -0,0 +1,58 @@
+from radix_sort import radixsort
+from random import shuffle
+import pytest
+
+
+def test_randoms():
+    x = range(100)
+    randoms = x[:]
+    shuffle(randoms)
+    radixsort(randoms)
+    y = randoms
+    assert x == y
+
+
+def test_empty():
+    x = []
+    y = x
+    radixsort(x)
+    assert x == y
```

Comment:
There is no way this can *not* work.  Both x and y point at the same list object.  Since your sort happens in-place, the set up for this test ensures that x will *always* be equal to y.